### PR TITLE
refactor(system-velodyne-monitor): rework parameters

### PR DESCRIPTION
**Pull request by karishma1911**
## Description

Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371 for the system_velodyne package.

1. Remove the default value from the source code in order to ensure all parameter values are passed from the parameter files.
2. Create the schema

## Tests performed

Not applicable.

## Effects on system behavior
Nothing.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by ito-san — APPROVED** [review]
LGTM

**Comment by ito-san** [comment]
@karishma1911 Thank you for te PR, but build errors occurred. Could you please fix it?
image

File: system/velodyne_monitor/src/velodyne_monitor.cpp [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
-  timeout_ = declare_parameter("timeout", 0.5);
-  ip_address_ = declare_parameter("ip_address", "192.168.1.201");
-  temp_cold_warn_ = declare_parameter("temp_cold_warn", -5.0);
-  temp_cold_error_ = declare_parameter("temp_cold_error", -10.0);
-  temp_hot_warn_ = declare_parameter("temp_hot_warn", 75.0);
-  temp_hot_error_ = declare_parameter("temp_hot_error", 80.0);
-  rpm_ratio_warn_ = declare_parameter("rpm_ratio_warn", 0.80);
-  rpm_ratio_error_ = declare_parameter("rpm_ratio_error", 0.70);
+  timeout_ = declare_parameter<double>("timeout");
+  ip_address_ = declare_parameter<string>("ip_address");
```
Comment (satoshi-ota) [inline system/velodyne_monitor/src/velodyne_monitor.cpp:36]:
```suggestion
  ip_address_ = declare_parameter<std::string>("ip_address");
```

Comment (satoshi-ota) [reply system/velodyne_monitor/src/velodyne_monitor.cpp:36]: @karishma1911

Comment (karishma1911) [reply system/velodyne_monitor/src/velodyne_monitor.cpp]: > @karishma1911

I fixed above issue, can you please go through this.
Thank you

**Comment by karishma1911** [comment]
> @karishma1911 Thank you for te PR, but build errors occurred. Could you please fix it? image

I fixed it..
Thank you

**Comment by satoshi-ota** [comment]
@karishma1911 Thanks. Please remove unnecessary commits :pray:

**Comment by karishma1911** [comment]
> @karishma1911 Thanks. Please remove unnecessary commits 🙏

sorry for inconvenience, after rebasing with main branch, those uncommited changes are pushed..
Please refer this PR on the same module and changes.https://github.com/autowarefoundation/autoware.universe/pull/5667
I will close this PR

**Pull request closed without merging**